### DISCUSSION TITLE
tensorflow.python.framework.errors_impl.DataLossError: not an sstable (bad magic number)

### DISCUSSION THREAD

**Discussion by hiyassat** [coqui-ai/TTS#897] [General]
i just got a new machine with Nvidia GPU 3090, Nvidia 3090 work fine on 
pip install --user nvidia-tensorflow[horovod] --use-feature=2020-resolver 
TensorFlow 15 
unfortunately this version of TensorFlow does not support tflite 
in order to export tflte model I tried to export the model on another machine that has deepspeech 9.3 
when I tried to export the model I got the error 

 this = _pywrap_tensorflow_internal.new_CheckpointReader(filename)
tensorflow.python.framework.errors_impl.DataLossError: not an sstable (bad magic number)

here is how I installed deepspeech 
https://discourse.mozilla.org/t/problems-training-on-rtx3080/68966/8
https://discourse.mozilla.org/t/problems-training-on-rtx3080/68966/8
https://github.com/NVIDIA/tensorflow#install
https://stackoverflow.com/questions/56431461/how-to-remove-cuda-completely-from-ubuntu
3090 for me only works with Cuda 11.1 and cudnn 8+ using the Nvidia’s TF 1.15
Install Cuda 11.1 and cudnn 8+ 
Install  Nvidia’s TF 1.15 from here
https://github.com/NVIDIA/tensorflow#install

conda create -n deepspeech python=3.6
conda activate deepspeech
  git clone --branch v0.9.3 https://github.com/mozilla/DeepSpeech
   cd DeepSpeech
   pip3 install --upgrade pip==20.2.2 wheel==0.34.2 setuptools==49.6.0
   pip3 uninstall numpy
   pip3 install numpy==1.18.5
   pip3 install --upgrade -e .
   sudo apt-get install python3-dev
   pip3 uninstall tensorflow
pip install --user nvidia-pyindex
pip install --user nvidia-tensorflow[horovod] --use-feature=2020-resolver 
 sh /home/ubuntu/DeepSpeech/bin/run-ted-h.sh

how I can export this model to tflite one

**Comment by erogol** [comment]
Wrong place
